### DISCUSSION TITLE
Running SQLAlchemy test suite with Oracle database

### DISCUSSION THREAD

**Discussion by savansonii** [sqlalchemy/sqlalchemy#7867] [Usage Questions]
Hi,

I want to know how sqlalchemy test suite handles the test setup for individual database, as I was executing the test suite for oracle database? There are some database dependencies from each database So how does the set up methods(database / table creation methods) are written.
Are they written individually for each database in dialect ?
Or they are written at some generic location?

**Comment by CaselIT** [comment]
Hi,

Please refer to https://github.com/sqlalchemy/sqlalchemy/blob/main/README.unittests.rst
